### DISCUSSION TITLE
[V3] Alpine Dropdown clips in viewbox (screencast version)

### DISCUSSION THREAD

**Discussion by xnasa** [livewire/livewire#8645] [1. Help]
Thanks for this drop down example.  When using this for table row action menu it displays and works well

![image](https://github.com/user-attachments/assets/5f665133-51f1-4487-b256-49e87a09ddcc)

However in the case where there is only one or a few rows the menu clips and is cut off at the containing table.

![image](https://github.com/user-attachments/assets/0415958c-af00-4ff0-ae2f-0e78a180fad8)

This will occur with only one line of data or at the edge of the table container when the menu is larger than the space available.
In my case, the table is being use as requests to process. The count can start with enough room to display the menu but eventually the user completes their work and the remaining list in the table shortens to one left. At that point the menu is cut off.

I can of course min height the whole table, but that leave a large white space with nothing going on in it.  (the table is a livewire component on the page)

Is there a way to change the x-menu elements to remain visible outside the table it is in.

The code is the https://livewire.laravel.com/screencast/data-tables/adding-row-actions repository.

**Comment by joshhanley** [comment]
@xnasa you probably need to remove overflow-hidden (or a class similar to that) from the table, otherwise the dropdown will be cut off like that.

Hope this helps!

**Comment by xnasa** [comment]
Yes the Table is wrapped by `<div class="relative overflow-x-auto">`
I removed the overflow directive and indeed the menu appeared correctly.  However that breaks the x scroll for narrow displays.
So I tried `overflow-visible overflow-x-scroll` or `overflow-y-visible overflow-x-scroll`  but it appears that the `-x-scroll` or `-x-auto` doesn't work in only the x direction.  the whole window clips with scroll even when only `-x-` directive is given.

So you are correct.  I'll mark you answer correct but I'd like to leave this open for a little bit longer to get a possible discussion on what it is the correct operational solution.  Is there a tailwinds css directive that will get it to work correctly or something updated in the Alpine UI file that will fix it, all while keeping the x scroll.
